### DISCUSSION TITLE
Searchable select

### DISCUSSION THREAD

**Discussion by kobe-ra** [livewire/livewire#3180] [4. General]
hello.

i'm moving from vue to livewire, but the only thing that hamper me is that im missing a searchable select package.

there is [livewire select](https://github.com/asantibanez/livewire-select) but the functionality isn't it.

[vue multiselect](https://vue-multiselect.js.org/) is what im aiming for.

functionality that i need is to give parameters
- list - collection of data, to fill the select options
- model - chosen option would be wired/bound to this variable
- title - list is collection of models, for example somewhere i want to show name, somewhere surname, this would also be the filter by value

i tried using select2, which works just fine. code similar to [this](https://www.positronx.io/how-to-create-select2-dropdown-in-laravel-livewire/) gave me a lot of functionality, but when i need to change model variable value in livewire component, i would need to emit some javascript event, which would get real ugly really fast. secondly, it is not bound to the model per se.

i am thinking of developing a package with desired functionality, but i cannot really imagine how to do it.

is there any advice, or is anyone possible already developing such package? maybe it even might become part of livewire itself later in the future? i feel like it's commonly used thing, yet so hard to do easily

**Comment by mithicher** [comment]
Hope this helps!
https://gist.github.com/mithicher/3633fd2c98dea98e2c4cde87e1f497e1

Check the original video
https://www.youtube.com/watch?v=L_f9gNaSCRE

**Comment by ph7jack** [comment]
You can use this package
https://livewire-wireui.com/docs/select

**Reply by kobe-ra** [reply]
this looks so good, thank you for letting me know it exists! it for sure deserves more than 55 stars, but i suppose more people need to discover it.

however, i'm currently looking for sth i could use in a large project which uses bootstrap and vue.

in documentation i see tailwind is not required as wireui has its own styles. alpine seems required no exception. i am not hundred percent sure, but as far as i know alpine fights with vue and should not be used in same projects. maybe i need to give it a shot anyways, but im afraid of this part.
